### ISSUE TITLE
check-redis-info.rb does not give any output when failing to connect

### ISSUE THREAD

**Issue by majormoses** [sensu-plugins/sensu-plugins-redis#39]
I discovered this when doing ruby 2.4.1 testing:
```
check-redis-info.rb -h 1.1.1.1 -K evicted_keys -V 0
```

There is no output at all, though it appears that it did fail:
```
$ echo $?
1
```

This is surprising as it looks like it should be throwing an exception.

**Comment by majormoses** [comment]
I validated the same behavior on 2.2.5 so the regression has been around for a while...

**Comment by majormoses** [comment]
it looks like some of them do have better error checking:
```
$ check-redis-memory.rb -h 1.1.1.1 -w 1048576 -c 1572864
RedisChecks WARNING: Could not connect to Redis server on 1.1.1.1:6379
```

**Comment by majormoses** [comment]
It looks like it should be using unknown there rather than message.

**Comment by Evesy** [comment]
Seems like a quick fix for this particular check as you mentioned

```
message "Could not connect to Redis server on #{config[:host]}:#{config[:port]}"
exit 1
```

Can just become:

```
unknown "Could not connect to Redis server on #{config[:host]}:#{config[:port]}"
```

However other checks, i.e. https://github.com/sensu-plugins/sensu-plugins-redis/blob/master/bin/check-redis-memory.rb, are using a mixture of warning/error exit codes for Redis connection timeouts (Some providing the choice).

I personally think it'd be good if there was a standard exit status for connection timeouts/failures (with the option to override if users so desire). My preference would be to exit unknown for erroneous Redis failures with the exception of `check-redis-ping.rb` which should exit with critical (As it's sole purpose is to check Redis is up & alive).

@majormoses Do you have any thoughts on whether this is necessary or not before I put together a PR for either scenario?

**Comment by majormoses** [comment]
What you propose makes total sense to me and I align with your thoughts, this was previously discussed in https://github.com/sensu-plugins/sensu-plugins-redis/pull/5 but due to lack of responsiveness we closed it out. I would welcome these becoming more standard, and completely agree that ones that's sole purpose is to connect and verify uptime should be criticial rather than unknown

**Comment by majormoses** [comment]
I'd certainly take a PR to standardize them even if it is a breaking change.

**Comment by majormoses** [comment]
fixed in #40 

released: https://rubygems.org/gems/sensu-plugins-redis/versions/2.0.0
